test/lint.js

```diff
@@ -155,17 +153,23 @@ const main = (files) => {
     );
   }
 
-  if (errors.length) {
+  const filesWithErrors = Object.keys(errors).length;
+
+  if (filesWithErrors) {
     console.error('');
     console.error(
       chalk`{red Problems in {bold ${pluralize('file', filesWithErrors)}}:}`,
     );
-    for (let error of errors) {
-      console.error(error);
+
+    for (const [fp, errorMsgs] of Object.entries(errors)) {
+      console.error(chalk`{red.bold ✖ ${fp}}`);
+      for (const error of errorMsgs) {
+        console.error(error);
+      }
     }
   }
 
-  return !!errors.length;
+  return !!errors.fileCount;
```

Comment:
I don't see the `fileCount` property being set above.

How about just:

```suggestion
  return filesWithErrors > 0;
```